eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/mob/ButcherCommand.java

```diff
@@ -45,7 +45,7 @@ void execute(@Context Player player, @Arg(ButcherArgument.KEY) int chunks) {
 
     @Execute
     @DescriptionDocs(description = "Kills specified mob in specified chunks around you", arguments = "<chunks> <mobType>")
-    void execute(@Context Player player, @Arg(ButcherArgument.KEY) int chunks, @Arg MobEntity mobEntity) {
+    void execute(@Context Player player, @Arg(ButcherArgument.KEY) int chunks, @Arg("mobType") MobEntity mobEntity) {
```

Comment:
The `@Arg("mobType")` change is probably useless because of [MobEntity](https://github.com/EternalCodeTeam/EternalCore/blob/c5a41ee690e668199481c6764a942149c382aacf/eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/mob/MobEntity.java) is an own type used only in EternalCore.